### PR TITLE
Unit tests for load indicators and search for pages and models.

### PR DESCRIPTION
**Pull request by bryan-gilbert**
More tests to come.

### CODE REVIEW

File: ppr-ui/tests/unit/SearchSerial.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    })
+    vm = wrapper.vm
+  })
+
+  afterEach( (): void => {
+    wrapper.destroy()
+  })
+
+  it('Test flagging feature one', (): void => {
+    expect(vm.$el.querySelector('#mockSerachResultsPage').textContent).toContain('Personal Property Registry')
```
Comment (WalterMoar) [inline ppr-ui/tests/unit/SearchSerial.spec.ts]: Typo?

File: ppr-ui/package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "vue-cli-service test:unit",
     "test:unit:cov": "vue-cli-service test:unit --coverage",
+    "test:one": "node node_modules/jest/bin/jest.js tests/unit/sample.spec.ts",
```
Comment (jguertin) [inline ppr-ui/package.json]: Would rather not see this, I only see it as useful if I edit it, and the IDE is perfectly good for running an individual test.  That's my vote, anyway.

**Review by jguertin — APPROVED** [review]
Just one comment, with my vote.  Will leave final decision to you/others.

File: ppr-ui/tests/unit/SearchSerial.spec.ts (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-ui/tests/unit/SearchSerial.spec.ts]: thanks

File: ppr-ui/package.json (hunk 2 above)
Comment (bryan-gilbert) [reply ppr-ui/package.json]: fix

**Review by WalterMoar — APPROVED** [review]
Yay tests!

**Pull request merged**